src/myWallet/wallet.service.js

```
@@ -538,19 +538,49 @@ const addCustomTronDeriveAddress = async (mnemonic, customDerivePath) => {
   }
 };
 
+const addCustomBitcoinDeriveAddress = async (
+  mnemonic,
+  isSandbox,
+  customDerivePath,
+) => {
+  try {
+    debugger;
```

Biome (2.3.13)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)